<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgScriptEditor</class>
 <widget class="QDialog" name="DlgScriptEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QToolButton" name="btnOpen">
       <property name="toolTip">
        <string>Open script</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+O, Return</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSave">
       <property name="toolTip">
        <string>Save</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSaveAs">
       <property name="toolTip">
        <string>Save as...</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Shift+S</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnEditHelp">
       <property name="toolTip">
        <string>Edit script help</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRun">
       <property name="toolTip">
        <string>Run algorithm</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>F5</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnCut">
       <property name="toolTip">
        <string>Cut</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+X</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnCopy">
       <property name="toolTip">
        <string>Copy</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+C</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnPaste">
       <property name="toolTip">
        <string>Paste</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+V</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnUndo">
       <property name="toolTip">
        <string>Undo</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Z</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRedo">
       <property name="toolTip">
        <string>Redo</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Shift+Z</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSnippets">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ScriptEdit" name="editor"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScriptEdit</class>
   <extends>QTextEdit</extends>
   <header>processing.gui.ScriptEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnSave</tabstop>
  <tabstop>btnSaveAs</tabstop>
  <tabstop>btnEditHelp</tabstop>
  <tabstop>btnRun</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
